pmlc/dialect/pxa/transforms/tile.h

```diff
@@ -0,0 +1,11 @@
+// Copyright 2019, Intel Corporation
+
+#pragma once
+
+#include "pmlc/dialect/pxa/ir/ops.h"
+
+namespace pmlc::dialect::pxa {
+
+void Tile(AffineParallelOp op, llvm::ArrayRef<int64_t> tileSizes);
```

Comment:
If tiling only works on `AffineParallelOp`, would it make sense to add a method to the `AffineParallelOp` itself?

Reply:
Maybe?  Not sure about this one, tiling feels too big to be in a helper function... Not quite today anyway.